test/pageobjects/cityWeatherPage.js

```diff
@@ -0,0 +1,22 @@
+import BasePage from "../../framework/page/BasePage.js";
+import Browser from "../../framework/browser/Browser.js";
+import { Label } from "../../framework/elements/index.js";
+
+class cityWeatherPage extends BasePage {
+    constructor() {
+        super(new Label('//div[contains(@class, "card-header")]'), 'cityPage');
+
+        this.cityName = new Label('//h1[@class="header-loc"]', 'city name');
+    }
+
+    async displayCityName() {
```

Comment:
functions that returns us bool we names form "is"

Reply:
Resolved